doc/source/structures/misc/gui.rst

```diff
@@ -208,13 +208,6 @@ following hierarchy:
     :attr:`ONTOGGLE`                      :struct:`KOSDelegate` (:struct:`Boolean`)  Your function called whenever the button's PRESSED state changes.
     ===================================== ========================================== =============
 
-.. note::
-
-    Reading the value of the :attr:`PRESSED` suffix will tell you if the button is pressed in (true)
-    or released (false).  But be aware that when :attr:`TOGGLE` is false, then the button will
-    remain pressed-in until such a time as your script detects that it has been pressed (so that
-    way the button won't press in-and-out too quickly for your script to notice).
-
```

Comment:
In looking at fixing issue #2022, I realized that actually instead of fixing the formatting of this section I can just remove it entirely because it's redundant with what's being said in each of the suffixes.